composer/datasets/cifar10.py

```diff
@@ -36,6 +40,54 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHpara
                 memory_format=self.synthetic_memory_format,
             )
 
+        elif self.use_ffcv:
+            try:
+                from ffcv.fields.decoders import IntDecoder, SimpleRGBImageDecoder
+                from ffcv.loader import Loader, OrderOption
+                from ffcv.transforms import (Convert, Cutout, RandomHorizontalFlip, RandomResizedCrop, RandomTranslate,
+                                             ToDevice, ToTensor, ToTorchImage)
+                from ffcv.transforms.common import Squeeze
+            except ImportError:
+                raise ImportError(
+                    textwrap.dedent("""\
+                    Composer was installed without ffcv support.
+                    To use ffcv with Composer, run `pip install mosaicml[ffcv]`"""))
+
+            if self.datadir is None:
+                raise ValueError("datadir is required if use_synthetic is False")
+
+            cifar10_ffcv_mean = [125.307, 122.961, 113.8575]
```

Comment:
Could you move this variable up so it is shared across the FFCV and non-ffcv codepaths?

Reply:
Done